src/components/DashboardOptions/DashboardOptions.tsx

```diff
@@ -1,15 +1,44 @@
 import Dropdown from 'react-bootstrap/Dropdown';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import styles from './DashboardOptions.module.css';
 import DropdownToggle from 'react-bootstrap/esm/DropdownToggle';
 import { ButtonGroup } from 'react-bootstrap';
+import { apiConfig, ApiError } from '../../api/ApiConfig';
+import { CurrentUserApi } from '@codecharacter-2023/client';
+import Toast from 'react-hot-toast';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { user } from '../../store/User/UserSlice';
+import { isTourResetChanged } from '../../store/DailyChallenge/dailyChallenge';
 
 interface dashboardoptions {
   image?: JSX.Element;
   onLogout?: React.MouseEventHandler<HTMLElement>;
 }
 
 const DashboardOptions = (props: dashboardoptions): JSX.Element => {
+  const currentUserApi = new CurrentUserApi(apiConfig);
+  const navigate = useNavigate();
+  const User = useAppSelector(user);
+
+  const dispatch = useAppDispatch();
+
+  const resetTutorials = () => {
+    currentUserApi
+      .updateCurrentUser({
+        name: User.name,
+        country: User.country,
+        college: User.college,
+        updateTutorialLevel: 'RESET',
+      })
+      .then(() => {
+        navigate('/dashboard');
+        dispatch(isTourResetChanged(true));
```

Comment:
Also make isTourOver false here to open that model
```suggestion
        dispatch(isTourOverChanged(false));
        dispatch(isTourResetChanged(true));
```